config/core/test-configs.yaml

```diff
@@ -1653,6 +1693,19 @@ device_types:
             - 'tinyconfig'
             - 'kvm_guest.config'
 
+  x86-simatic-ipc227e:
```

Comment:
This one is failing to boot, apparently it's a problem with the LAVA configuration as it can't do GRUB:
https://lava.ciplatform.org/scheduler/job/610420